std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/DoubleStorage.java

```diff
@@ -40,7 +36,7 @@
 
 /** A column containing floating point numbers. */
 public final class DoubleStorage extends Storage<Double>
```

Comment:
Why are these `XyzStorage` implementations `public`? We need them to implement the storage of _memory layout_ effectively, but otherwise they are an implementation detail. Once a _builder_ builds a storage it should be `Storage<Double>` and the actual implementation shall be package `private`.

Reply:
Yes that is the aim but currently they are leaked everywhere.